cmd/jobs.go

```diff
@@ -85,6 +85,15 @@ var jobsPurgeCmd = &cobra.Command{
 		if len(args) != 1 {
 			return cmd.Help()
 		}
+		if err := config.Setup(cfgFile); err != nil {
+			return err
+		}
+		if config.FsURL().Scheme == config.SchemeSwift ||
+			config.FsURL().Scheme == config.SchemeSwiftSecure {
+			if err := config.InitSwiftConnection(config.GetConfig().Fs); err != nil {
+				return err
+			}
+		}
```

Comment:
Why do we need a connection to swift for purging old jobs?

Reply:
The reason is that `instance.GetFromCouch` method calls the `instance.MakeVFS()`. If swift connection is not initialized, the stack panics (https://github.com/cozy/cozy-stack/blob/4c1765674e2c09988bd728927577e5fc4f158c76/pkg/instance/instance.go#L541)